### PR TITLE
Fix some inaccuracies in non-cluster hosts docs

### PR DESCRIPTION
**Pull request by rene-dekker**

### CODE REVIEW

File: calico-cloud/_includes/components/EnvironmentFile.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ export default function EnvironmentFile(props) {
         </Admonition>
       )}
       <p>
-        Sample <code>EnvironmentFile</code> - save to <code>/kubernetes/calico/calico.env</code>
+        Sample <code>EnvironmentFile</code> - save to <code>/etc/calico/calico.env</code>
```
Comment (ctauchen) [inline calico-cloud/_includes/components/EnvironmentFile.js:46]: Should this change also go in CC18?
Seems to go in a lot of places:
image

**Review by ctauchen — COMMENTED** [review]
Just the one question. Otherwise LGTM

File: calico-cloud/_includes/components/EnvironmentFile.js (hunk 1 above)
Comment (rene-dekker) [reply calico-cloud/_includes/components/EnvironmentFile.js:46]: Nice catch! Pushed a commit.

**Comment by ctauchen** [comment]
LGTM. Ready to merge?

**Pull request merged**